helpers/python.ts

```diff
@@ -70,6 +70,20 @@ const getAdditionalDependencies = (
       });
       break;
     }
+    case "qdrant": {
```

Comment:
@leehuwuj oops was qdrant missing?

Reply:
Yes, we missed that when we added the qdrant vector store!